bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gems/CanopusFeature.java

```diff
@@ -250,166 +135,23 @@ public void draw(Graphics g, TpeImageInfo tii) {
-        if (rangeMode == mode) return;
-        rangeMode = mode;
-        _redraw();
-    }
-
-    public void handleModeChange(TpeMode mode, Option<Object> arg) {
-        if ((mode != TpeMode.CREATE) || arg.isEmpty()) {
-            setRangeDisplayMode(RangeDisplayMode.both);
-            return;
-        }
-
-        Object value = arg.getValue();
-        if (!(value instanceof TpeGuidePosCreatableItem)) {
-            setRangeDisplayMode(RangeDisplayMode.both);
-            return;
-        }
-
-        TpeGuidePosCreatableItem item = (TpeGuidePosCreatableItem) value;
-        GuideProbe guider = item.getGuideProbe();
-        if (guider == Canopus.Wfs.cwfs1) {
-            setRangeDisplayMode(RangeDisplayMode.probe1);
-        } else if (guider == Canopus.Wfs.cwfs2) {
-            setRangeDisplayMode(RangeDisplayMode.probe2);
-        } else {
-            setRangeDisplayMode(RangeDisplayMode.both);
-        }
-    }
-
-    private static boolean containsTarget(TargetEnvironment env, GuideProbe guider, SPTarget target) {
-        final Option<GuideProbeTargets> gtOpt = env.getPrimaryGuideProbeTargets(guider);
-        return gtOpt.exists(gt -> gt.containsTarget(target));
-    }
-
-    public void handleDragStarted(Object dragObject, ObsContext context) {
-        if (!(dragObject instanceof SPTarget)) return;
-
-        SPTarget target = (SPTarget) dragObject;
-        TargetEnvironment env = context.getTargets();
-        if (env == null) {
-            setRangeDisplayMode(RangeDisplayMode.both);
-            return;
-        }
-
-        if (containsTarget(env, Canopus.Wfs.cwfs1, target)) {
-            setRangeDisplayMode(RangeDisplayMode.probe1);
-        } else if (containsTarget(env, Canopus.Wfs.cwfs2, target)) {
-            setRangeDisplayMode(RangeDisplayMode.probe2);
-        } else {
-            setRangeDisplayMode(RangeDisplayMode.both);
-        }
-    }
-
-    public void handleDragStopped(ObsContext context) {
-        setRangeDisplayMode(RangeDisplayMode.both);
-    }
-
-    private static class ProbeRangeIcon implements Icon {
-        private final Orientation[] slants;
-
-        ProbeRangeIcon(Orientation... slants) {
-            this.slants = slants;
-        }
-
-        public int getIconWidth()  { return 18; }
-        public int getIconHeight() { return 18; }
-
-        public void paintIcon(Component c, Graphics g, int x, int y) {
-            Graphics2D g2d = (Graphics2D) g;
-            g2d.setColor(Color.black);
-            g2d.fill(new Rectangle2D.Double(1, 1, 16, 16));
-
-            Paint origPaint = g2d.getPaint();
-            for (Orientation slant : slants) {
-                Paint p = createProbeRangeKeyPaint(g2d, slant);
-                g2d.setPaint(p);
-                g2d.fill(new Rectangle2D.Double(1, 1, 16, 16));
-            }
-            g2d.setPaint(origPaint);
-        }
-    }
-
     @Override
-    public Option<Component> getKey() {
```

Comment:
This was just a legend for the CWFS1 and 2 probe ranges; thus, it is no longer needed.